install-steps.md

```diff
@@ -554,7 +553,34 @@ sudo cp ./oc /usr/local/bin/oc
 oc adm release extract --registry-config "${pullsecret_file}" --command=$cmd --to "${extract_dir}" ${RELEASE_IMAGE}
 ~~~
 
+### Create RHCOS images cache (Optional)
+
+The installer needs to download two images, the RHCOS image used by the bootstrap VM and the RHCOS image used by the installer to provision the different nodes. Image caching is specially useful when running the installer on a network with limited bandwidth.
+
+When running the installer on a network with limited bandwidth if the RHCOS images download takes more than 15-20 minutes, the installer will timeout, having these images cached on a local `httpd` will help in such scenarios.
 
+The following steps will install `httpd` and download the required images:
```

Comment:
I would change this to:
The following steps install `httpd` and download the required images onto the provision host: 

@mvazquezc - My only issue with the instructions below is that its not a pod matching what we do today with the Ansible installer. What do you think if we do the following instead? Also FYI, you use `yum` but really should use `dnf` across the board since its RHEL8.1

```
# Install podman
sudo dnf install -y podman

# Create a directory to store the bootstraposimage and clusterosimage
mkdir /home/kni/rhcos_image_cache
# Set the appropriate SELinux context for the newly created directory
sudo semanage fcontext -a -t httpd_sys_content_t "/home/kni/rhcos_image_cache(/.*)?"
sudo restorecon -Rv rhcos_image_cache/
export RHCOS_OPENSTACK_URI=$(curl -s -S https://raw.githubusercontent.com/openshift/installer/$COMMIT_ID/data/data/rhcos.json  | jq .images.openstack.path | sed 's/"//g')
# Get the uri for the RHCOS image that will be deployed on the bootstrap vm
export RHCOS_QEMU_URI=$(curl -s -S https://raw.githubusercontent.com/openshift/installer/$COMMIT_ID/data/data/rhcos.json  | jq .images.qemu.path | sed 's/"//g')
# Get the path where the images are published
export RHCOS_PATH=$(curl -s -S https://raw.githubusercontent.com/openshift/installer/$COMMIT_ID/data/data/rhcos.json | jq .baseURI | sed 's/"//g')
# Get the SHA hash for the RHCOS image that will be deployed on the bootstrap vm
export RHCOS_QEMU_SHA_UNCOMPRESSED=$(curl -s -S https://raw.githubusercontent.com/openshift/installer/$COMMIT_ID/data/data/rhcos.json  | jq -r '.images.qemu["uncompressed-sha256"]')
# Get the SHA hash for the RHCOS image that will be deployed on the nodes
export RHCOS_OPENSTACK_SHA_COMPRESSED=$(curl -s -S https://raw.githubusercontent.com/openshift/installer/$COMMIT_ID/data/data/rhcos.json  | jq -r '.images.openstack.sha256')
# Download the images and place them in the  /home/kni/rhcos_image_cache directory
sudo curl -L ${RHCOS_PATH}${RHCOS_QEMU_URI} -o /home/kni/rhcos_image_cache
sudo curl -L ${RHCOS_PATH}${RHCOS_OPENSTACK_URI} -o /home/kni/rhcos_image_cache

# Create pod
podman run -d --name rhcos_image_cache \
-v /home/kni/rhcos_image_cache:/var/www/html \
-p 8080:8080/tcp \
registry.centos.org/centos/httpd-24-centos7:latest


```